### PR TITLE
Package updates 2023 10

### PR DESCRIPTION
**Pull request by JackKelly-Bellroy**

### CODE REVIEW

File: aws-arn.cabal [hunk 1]
```diff
@@ -38,7 +38,7 @@ extra-source-files:
   README.md
 
 tested-with:
-  GHC ==8.6.5 || ==8.8.4 || ==8.10.7 || ==9.0.2 || ==9.2.1
+  GHC ==8.10.7 || ==9.0.2 || ==9.2.5 || ==9.4.5 || ==9.6.2
```
Comment (Tristano8) [inline aws-arn.cabal]: Might be worth a wiki/handbook/asana template entry on how we do these

File: aws-arn.cabal [hunk 2]
```diff
@@ -51,11 +51,11 @@ common opts
 
 common deps
   build-depends:
-    , base             >=4.12   && <4.18
+    , base             >=4.12   && <4.19
     , deriving-compat  >=0.5.10 && <0.7
     , profunctors      >=5.0    && <5.7
     , tagged           ^>=0.8
-    , text             ^>=1.2.3 || ^>=2.0
+    , text             ^>=1.2.3 || >=2.0 && <2.1 || ^>=2.1
```
Comment (Tristano8) [inline aws-arn.cabal:58]: Why this and not `^>=1.2.3 || >=2.0 && <2.2` ?

**Review by Tristano8 — APPROVED** [review]

File: aws-arn.cabal (hunk 2 above)
Comment (JackKelly-Bellroy) [reply aws-arn.cabal:58]: Beats me, but that's what `cabal-fmt` turned it into.

File: aws-arn.cabal (hunk 1 above)
Comment (JackKelly-Bellroy) [reply aws-arn.cabal]: OK

Comment (JackKelly-Bellroy) [reply aws-arn.cabal]: https://github.com/bellroy/haskell/pull/2486

**Pull request merged**